tests/unittests/odm_tests.py

```
@@ -109,3 +109,85 @@ def test_get_indexed_values(self):
+                "_cls": "UnknownType1",
+                "level2": {
+                    "_cls": "UnknownType2",
+                    "value": "nested_value",
+                },
+            },
+        }
+
+        result = foou.sanitize_unknown_embedded_docs(data)
+
+        # Both levels should be preserved
+        self.assertIn("level1", result)
```

Ruff (0.14.10)

Use a regular `assert` instead of unittest-style `assertIn`

Replace `assertIn(...)` with `assert ...`

(PT009)